app/src/main/java/com/example/henzoshimada/feeltrip/User.java

```diff
@@ -0,0 +1,48 @@
+package com.example.henzoshimada.feeltrip;
+
+import io.searchbox.annotations.JestId;
+
+/**
+ * Created by Brett on 2017-03-20.
+ */
+
+class User {
+    private String username;
+    private String password;
+
```

Comment:
Username and password is vital to what a user account needs, we'll also need a list of who the user is following. Can anyone think of anything else?